src/diffusers/pipeline_utils.py

```diff
@@ -664,6 +677,14 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         model = pipeline_class(**init_kwargs)
         return model
 
+    @staticmethod
+    def _get_signature_keys(obj):
```

Comment:
this function is very important - it defines what keyword args are modules and which are not cc @patil-suraj @pcuenca @anton-l